libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -84,6 +127,14 @@ public virtual IEnumerable<Dialog> GetDependencies()
             }
         }
 
+        /// <summary>
+        /// Called when a returning control to this dialog with an <see cref="ActionScopeResult"/>.
```

Comment:
Could be described differently?, same for the rest of the methods.

Reply:
removed the `a`, I think that was a typo causing some confusion in the interpretation